nautobot_golden_config/views.py

```diff
@@ -292,6 +311,42 @@ def get(self, request, pk, config_type):  # pylint: disable=invalid-name,too-man
                 + "@@"
                 + output[second_occurence + 2 :]
             )
+        elif config_type == "json_compliance":
+            compliance_objects = models.ConfigCompliance.objects.filter(device=device.id)
+            actual = {}
+            intended = {}
+            most_recent_time = datetime(1970, 1, 1, tzinfo=timezone.utc)
+            for obj in compliance_objects:
+                actual[obj.rule.feature.slug] = json.loads(obj.actual)
+                intended[obj.rule.feature.slug] = json.loads(obj.intended)
+                if obj.last_updated > most_recent_time:
+                    most_recent_time = obj.last_updated
+            config_details.backup_last_attempt_date = most_recent_time
```

Comment:
I have to think about this, what is the time a user would expect to be here?

Reply:
So my thought process was getting the most recent time out of all the different config compliance rule that were hit.  So we get the list, I set the time to 1970 just to make it an old time that we can update with most recent as we loop through the rules(compliance) objects.

Reply:
@itdependsnetworks, still have a concern with this?